bin/runex

```diff
@@ -22,4 +22,4 @@ if command -v accumulo > /dev/null 2>&1 ; then
   av_arg="-Daccumulo.version=`accumulo version | tail -n 1`"
 fi
 
-mvn -q exec:java -Dlog4j.configuration="file:./conf/log4j.properties" -Dexec.mainClass="org.apache.accumulo.examples.$main_class" $av_arg -Dexec.args="$main_args"
+mvn -X exec:java -Dlog4j.configuration="file:./conf/log4j.properties" -Dexec.mainClass="org.apache.accumulo.examples.$main_class" $av_arg -Dexec.args="$main_args"
```

Comment:
This should be -q by default.  I would be fine if this was added back in if users ran `./bin/runex -d <my class>`

Reply:
I forgot to remove this.  I opened up a separate issue #18